src/app/dashboard/settings/profile/components/EditProfile.tsx

```diff
@@ -1,37 +1,54 @@
+'use client'
 import avatars from '@public/avatars'
 import Image from 'next/image'
+import { useState } from 'react'
 import editProfile from 'public/edit-profile.png'
 import { AiOutlineCamera } from 'react-icons/ai'
-const EditProfile = () => {
+import useUserStore from '@/state/userStore'
+const EditProfile = ({ onclick }) => {
+  const { user } = useUserStore()
+  const [name, setName] = useState(user?.name)
```

Comment:
@Sheriff-Oladimeji can u use userStore for name and email. Also make sure every field is editable